socorro/unittest/external/postgresql/test_adi.py

```diff
@@ -62,13 +101,37 @@ def setUp(self):
                             channel,
                         ))
                         adi_count *= 2
-        cursor.execute('select count(*) from raw_adi')
-        count, = cursor.fetchone()
-        # We expect there to be 2 channels per every 2 platforms,
-        # per every 2 versions per every 2 products.
-        assert count == 2 * 2 * 2 * 2, count
+
+                product_version_id += 1
+                cursor.execute("""
+                    INSERT INTO product_versions
+                    (product_version_id, product_name, major_version,
+                     release_version, version_string, version_sort,
+                     build_date, sunset_date, featured_version,
+                     build_type, build_type_enum)
+                    VALUES (%s, %s, %s, %s, %s, %s, %s, %s, 't', %s, %s)
+                """, (
+                    product_version_id,
+                    product,
+                    version,
+                    version,
+                    version,
+                    '0000' + version.replace('.', ''),
+                    build_date,
+                    sunset_date,
+                    'release',
+                    'release',
+                ))
+
+        cursor.callproc('update_adu', [yesterday.date()])
```

Comment:
Nice, we're starting to test stored procedures in our integration tests, that's great!